proiect MDPA1/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;

```

Comment:
One file, one class. This is a standard rule in most programming languages to make the code more readable and the project easier to manage